test/dal/submissions.spec.ts

```diff
@@ -38,5 +59,77 @@ describe('Submissions DAL', () => {
         expect(err.message).to.equal(fakeError.message);
       }
     });
+
+    it('should create submission successfully', async () => {
+      sandbox.stub(pool, 'query').callsFake(async () => {
+        return Promise.resolve([createSuccessObj]);
+      });
+      const response = await create(createSuccessObj);
+      expect(response).to.equal(createSuccessObj);
+    });
+
+    it('should handle invalid latitude, longitude values', async () => {
+      sandbox.stub(pool, 'query').callsFake(async () => {
+        return Promise.reject(errInvalidArgs);
```

Comment:
What returns errInvalidArgs? pool.query or the submissions.create? We also don't need to test individual validation cases in the DAL logic, we can have tests for verifySubmission that ensure it's throwing errors when it should. The submisisons.create logic really only needs one test to confirm an error throw by verifySubmission handled as we expect.